tests/test_cases/complete_appnexus.py

```diff
@@ -4,7 +4,6 @@
 from functools import *
 import os
 from os import path
-import StringIO
```

Comment:
I don't see any stdlib module in Py3 that start with an uppercase, so I think this just has to be removed from the tests.